geowebcache/core/src/main/java/org/geowebcache/storage/DefaultStorageFinder.java

```diff
@@ -130,9 +130,9 @@ private void determineDefaultPrefix() {
             if (tmpDir != null) {
                 File temp = new File(tmpDir, "geowebcache");
                 logMsg =
-                        "Reverting to java.io.tmpdir "
-                                + this.defaultPrefix
-                                + " for storage. "
+                        "Reverting to java.io.tmpdir '"
+                                + temp
+                                + "' for storage. "
```

Comment:
`java.io.File.toString()` can return just a relative path, depending on what has been using to construct the instance. I'd use here `file.getAbsolutePath()` to be sure that the absolute path is printed.